File: Templates/CSharp/WebApplication/Model/MyEntity.cs
using System;
using Xtensive.Orm;

namespace $safeprojectname$.Model
{
	[HierarchyRoot]
	public class MyEntity : Entity
	{
		[Field, Key]
		public int Id { get; private set; }

		[Field(Length = 100)]
		public string Text { get; set; }


		public MyEntity()
			: base()
		{
		}

		public MyEntity(Session session)
			: base(session)
		{
		}
	}
}

File: Templates/VisualBasic/Console/Model/MyEntity.vb
Imports Xtensive.Orm

Namespace Model
    <HierarchyRoot>
    Public Class MyEntity 
        Inherits Entity
  
        Public Sub New()
            MyBase.New()
        End Sub

        Public Sub New(ByVal session As Session)
            MyBase.New(session)
        End Sub

        <Field, Key>
        Public Property Id As Integer

        <Field(Length := 100)>
        Public Property Text As String

    End Class
End NameSpace